.gitignore

```diff
@@ -30,6 +30,7 @@ package-lock.json
 *.tfstate.*
 .terraform.tfstate.lock.info
 .terraform.lock.hcl
+terraform
```

Comment:
Yes, you are correct :+1: I wanted to avoid modified files if somebody run `airy create` in the root of the repository (this will create a `terraform` directory in the root. But I will figure another way.